cmake/std/atdm/ats2/environment.sh

```diff
@@ -253,7 +257,9 @@ export F90=mpifort
 
 # Define function atdm_run_script_on_compute_node
 unset atdm_run_script_on_compute_node
-source $ATDM_SCRIPT_DIR/common/define_atdm_run_script_on_local_node.sh
+
+# TODO: Will this work be able to find the cuda runtime for targeting GPUs?
+source $ATDM_SCRIPT_DIR/common/define_run_on_lsf_compute_node_func.sh
```

Comment:
Should this use `ATDM_CONFIG_SCRIPT_DIR` instead?

Reply:
It could but other environments are experiencing the same "relative path issue" so it seems to make more sense to just make `ATDM_SCRIPT_DIR` an absolute path as well, since, AFAIK, this does not change the semantics of ATDM_SCRIPT_DIR.

Reply:
@e10harvey, I think it makes sense to make `ATDM_CONFIG_SCRIPT_DIR` the abs path and then use it consistently throughout and get rid of `ATDM_SCRIPT_DIR` unless there is a reason to keep `ATDM_SCRIPT_DIR`.